lib/src/components/NxTab/NxTab.scss

```diff
@@ -6,16 +6,32 @@
  */
  @import '../../scss-shared/nx-colors';
  @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
 
 .nx-tab {
-  border-bottom: 4px solid transparent;
+  border-bottom: 2px solid transparent;
```

Comment:
Looks like it's 1px except when active, then it's 2

Reply:
> Looks like it's supposed to be 1px except when active, then it's 2

Oh I see, you've got the active border on top of the tar bar border.  It looks like it's supposed to overlap it.

Reply:
> > Looks like it's supposed to be 1px except when active, then it's 2
> 
> Oh I see, you've got the active border on top of the tab bar border. It looks like it's supposed to overlap it.

https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/4430fc5bac08f24e0a5c477ecb69fb56bdd67850